.github/workflows/vs16-noexcept-ci.yml

```diff
@@ -0,0 +1,36 @@
+name: VS16-NoExcept-CI
```

Comment:
Was this intended?

Reply:
Yes. It is like what we have in the main branch. The convention is to use at least one distinct name per file (i.e., if you have k iml files, you should have at least k distinct names).

Reply:
Oh sorry, I was checking if you intended to add the file :)

Reply:
Yes. This file is not really part of the PR. I only really changed the error reporting, and only for the double type.